dotnet/src/Functions/Functions.OpenApi/Serialization/OpenApiTypeConverter.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// Provides functionality for converting OpenApi types - https://swagger.io/docs/specification/data-models/data-types/
+/// </summary>
+internal static class OpenApiTypeConverter
+{
+    /// <summary>
+    /// Converts the given parameter argument to a JsonNode based on the specified type.
+    /// </summary>
+    /// <param name="name">The parameter name.</param>
+    /// <param name="type">The parameter type.</param>
+    /// <param name="argument">The argument to be converted.</param>
+    /// <returns>A JsonNode representing the converted value.</returns>
+    public static JsonNode Convert(string name, string type, object argument)
+    {
+        Verify.NotNull(argument);
+
+        try
+        {
+            if (s_converters.TryGetValue(type, out var converter))
+            {
+                var result = converter(name, type, argument);
```

Comment:
It doesn't look like `name` and `type` are used in any of the converters. Should we remove them from here?

Reply:
Right, they are not used there. They were originally used, but then I refactored the code to return null instead of throwing an exception and forgot to remove them. Thanks, removed.